### ISSUE TITLE
URI::InvalidURIError: URI must be ascii only

### ISSUE THREAD

**Issue by tateisu** [tootsuite/mastodon#2119]
* this error should not retry
* "https://丼.upsilo.net/" should be converted to punycode domains

```
RemoteProfileUpdateWorker

URI::InvalidURIError: URI must be ascii only "https://\u4E3C.upsilo.net/system/accounts/avatars/000/000/001/original/7222040ab84610a9.png?1492538893"

13397, "<?xml version=\"1.0\"?>
\n<feed xmlns=\"http://www.w3.org/2005/Atom\" xmlns:thr=\"http://purl.org/syndication/thread/1.0\" xmlns:activity=\"http://activitystrea.ms/spec/1.0/\" xmlns:poco=\"http://portablecontacts.net/spec/1.0\" xmlns:media=\"http://purl.org/syndication/atommedia\" xmlns:ostatus=\"http://ostatus.org/schema/1.0\" xmlns:mastodon=\"http://mastodon.social/schema/1.0\">
\n <id>https://丼.upsilo.net/users/upsilon.atom</id>
\n <title>upsilon</title>
\n <subtitle>https://twitter.com/kim_upsilon</subtitle>
\n <updated>2017-04-18T19:21:47Z</updated>
\n <logo>https://丼.upsilo.net/system/accounts/avatars/000/000/001/original/7222040ab84610a9.png?1492538893</logo>
\n <author>
\n <id>https://丼.upsilo.net/users/upsilon</id>
\n <activity:object-type>http://activitystrea.ms/schema/1.0/person</activity:object-type>
\n <uri>https://丼.upsilo.net/users/upsilon</uri>
\n <name>upsilon</name>
\n <email>upsilon@丼.upsilo.net</email>
\n <summary>https://twitter.com/kim_upsilon</summary>
\n <link rel=\"alternate\" type=\"text/html\" href=\"https://丼.upsilo.net/@upsilon\"/>
\n <link rel=\"avatar\" type=\"image/png\" media:width=\"120\" media:height=\"120\" href=\"https://丼.upsilo.net/system/accounts/avatars/000/000/001/original/7222040ab84610a9.png?1492538893\"/>
\n <link rel=\"header\" type=\"\" media:width=\"700\" media:height=\"335\" href=\"https://丼.upsilo.net/headers/original/missing.png\"/>
\n <poco:preferredUsername>upsilon</poco:preferredUsername>
\n <poco:displayName>upsilon</poco:displayName>
\n <poco:note>&lt;a href=\"https://twitter.com/kim_upsilon\" rel=\"nofollow noopener\" target=\"_blank\"&gt;&lt;span class=\"invisible\"&gt;https://&lt;/span&gt;&lt;span class=\"\"&gt;twitter.com/kim_upsilon&lt;/span&gt;&lt;span class=\"invisible\"&gt;&lt;/span&gt;&lt;/a&gt;</poco:note>
\n <mastodon:scope>public</mastodon:scope>
\n </author>
\n <link rel=\"alternate\" type=\"text/html\" href=\"https://丼.upsilo.net/@upsilon\"/>
\n <link rel=\"self\" type=\"a
..., false
```

* * * *

- [x ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ x] This bug happens on a v1.2

**Comment by tateisu** [comment]
丼.upsilo.net has died. https://m.upsilo.net/@upsilon/1
he changed instance name.

**Comment by upsilon** [comment]
I'm owner of 丼.upsilo.net

丼.upsilo.net was on `master` branch with some workaround fix for #1884 (HEAD: upsilon@c90da1dca073121f905f901c44a31903c787144d).
Also added `LOCAL_DOMAIN=丼.upsilo.net` (not `LOCAL_DOMAIN=xn--uiq.upsilo.net`) to `.env.production`.
